.travis.yml

```diff
@@ -27,7 +27,9 @@ matrix:
     - php: 5.6
       env: DEPENDENCIES=dev
     - php: 7.0
+      env: SYMFONY_PHPUNIT_VERSION=6.5
```

Comment:
Is this really needed?

Reply:
My intention was to ensure that we maintain compatibility with PHPUnit 6. Without that setting, the PHPUnit bridge would execute PHPUnit 5.7.

https://github.com/symfony/phpunit-bridge/blob/5d27d894072832dce51f73ef3043ce3dfe0f866c/bin/simple-phpunit#L21-L23

Reply:
What about adding a job for PHP 7.2 instead then?

Reply:
All right, I'll do that. May I pin the php 7.2 job to PHPUnit 6.5? PHPUnit bridge would run PHPUnit 6.3 otherwise.